server/build.gradle.kts

```diff
@@ -13,18 +16,22 @@ plugins {
 	jacoco
 }
 
+
 group = "pis-hello"
 version = "0.0.1-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_11
 
 repositories {
+	google()
 	mavenCentral()
 }
 
 dependencies {
+	implementation("androidx.camera:camera-camera2:1.0.0-alpha07")
```

Comment:
Why do we have android camera in our server dependencies?

Reply:
Dependencies deleted